### PR TITLE
SwiftLint & Rename Project without Spaces

### PR DESCRIPTION
**Pull request by Mackis**
- Renamed the project to not include spaces (this could lead to CI issues later)
- Added SwiftLint 
- Fixed all current listing errors and warnings 
- Added SwiftLint Github action

### CODE REVIEW

**Comment by Mackis** [comment]
Will be resolving conflicts

**Comment by rodericj** [comment]
Related. You’re is probably fresher.

File: SayTheirNames/Source/Controller/Base/BaseNibLoading.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//  UIViewController+STN.swift
+//  SayTheirNames
+//
+//  Created by evilpenguin on 5/31/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+public protocol BaseNibLoading {
```
Comment (hybridcattt) [inline SayTheirNames/Source/Controller/Base/BaseNibLoading.swift:11]: this file was removed in #39

File: SayTheirNames/Source/Controller/Home/HomeView.swift [hunk 2]
```diff
@@ -129,7 +129,7 @@ final class HomeView: UIView {
         
         let logo = UIImageView(image: UIImage(named: "logo"))
         let label = UILabel()
-        label.text = "SAY THEIR NAME"
+        label.text = "SayTheirName"
```
Comment (hybridcattt) [inline SayTheirNames/Source/Controller/Home/HomeView.swift:132]: accidental change here?

File: SayTheirNames/Source/Utils/Extentions/UIDevice+STN.swift [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+//
+//  Created by evilpenguin on 5/30/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - STNDevice
+
+public typealias STNDevice = UIDevice
```
Comment (hybridcattt) [inline SayTheirNames/Source/Utils/Extentions/UIDevice+STN.swift:13]: not sure this file is supposed to be added in this PR?

File: SayTheirNames/Source/Utils/Fonts/UIFont+Resource.swift [hunk 4]
```diff
@@ -0,0 +1,79 @@
+//
+//  FontResource.swift
```
Comment (hybridcattt) [inline SayTheirNames/Source/Utils/Fonts/UIFont+Resource.swift:2]: wondering why this file is added here? is it a duplicate?

File: SayTheirNames/Source/View/Home/Person/PersonCell.xib [hunk 5]
```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hybridcattt) [inline SayTheirNames/Source/View/Home/Person/PersonCell.xib:1]: same here

File: SayTheirNames/Source/Controller/Base/BaseNibLoading.swift (hunk 1 above)
Comment (Mackis) [reply SayTheirNames/Source/Controller/Base/BaseNibLoading.swift:11]: Let me merge with master again.

File: SayTheirNames/Source/View/Home/Person/PersonCell.xib (hunk 5 above)
Comment (Mackis) [reply SayTheirNames/Source/View/Home/Person/PersonCell.xib:1]: Merging with latest develop resolved this

File: SayTheirNames/Source/Controller/Home/HomeView.swift (hunk 2 above)
Comment (Mackis) [reply SayTheirNames/Source/Controller/Home/HomeView.swift]: Yes thanks for spotting.

File: SayTheirNames/Source/Controller/Base/BaseNibLoading.swift (hunk 1 above)
Comment (hybridcattt) [reply SayTheirNames/Source/Controller/Base/BaseNibLoading.swift:11]: didn't help 🤔  
would it be easier to re-apply the changes off the latest development, what do you think? if you do so, would be ideal to have renaming the project and adding SwiftLint as separate PRs

Comment (Mackis) [reply SayTheirNames/Source/Controller/Base/BaseNibLoading.swift]: I ended up removing the file. It was still hanging around in the folder but the project had no reference to it.

**Comment by hybridcattt** [comment]
Sorry I can’t have another look as I’m away for a while, please ping the iOS channel on slack - would be great to merge this asap 🙌👏

File: SayTheirNames/Source/View/Home/Person/PersonCell.xib (hunk 5 above)
Comment (hybridcattt) [reply SayTheirNames/Source/View/Home/Person/PersonCell.xib:1]: This file is still here 😅

**Pull request merged**